examples/08_05_multithreaded_tcp_server/src/main.rs

```diff
@@ -0,0 +1,26 @@
+// ANCHOR: main_func
+use async_std::net::{TcpListener, TcpStream};
+use async_std::task::spawn;
+
+#[async_std::main]
+async fn main() {
+    let listener = TcpListener::bind("127.0.0.1:7878").await.unwrap();
+
+    loop {
```

Comment:
Maybe we could keep using `for_each(spawn(handle_connection))` to keep the code change minimal and still utilize the Stream?